bot/config.py

```diff
@@ -35,7 +35,7 @@
 SUPPORT_SERVER = "https://discord.gg/7e9zKFr"
 
 
-COMMAND_PREFIX = os.getenv("COMMAND_PREFIX", ">>")
+COMMAND_PREFIX = os.getenv("COMMAND_PREFIX", ";")
```

Comment:
```suggestion
COMMAND_PREFIX = os.getenv("COMMAND_PREFIX", ">>")
```
why did you change the prefix again?